routes/libraries.js

```diff
@@ -21,232 +23,67 @@ const storage = multer.diskStorage({
-  res.render("libraries/new", {
-    title: "Creating a Library",
-    id: "NEED TO FIX",
-  });
-});
+router.route('/new')
+  .get(async (req, res) => {
+    // Render the new library form page
 
-router.route("/:id").post(async (req, res) => {
-  // Grab the library id
-  const libraryId = req.params.id;
-  // Grab the form data
-  const updateData = req.body;
-  // Using the similar data checking function as above. Maybe this should be made into a checker function?
-  if (!updateData || Object.keys(updateData).length === 0) {
-    return res
-      .status(400)
-      .json({ error: "There are no fields in the request body" });
-  }
-  /**
-   * Assumed Form data is of type:
-   * {
-   *  fullnessRating: number
-   *  genres: [array<string>]
-   * }
-   * This is using the error checking from before, using the helper functions for uniformity.
-   */
-  try {
-    updateData.fullness = validation.isValidNumber(
-      updateData.fullness,
-      "Fullness Rating"
-    );
-    updateData.genres = validation.checkStringArray(
-      updateData.genres,
-      "Genres Available"
-    );
-  } catch (e) {
-    return res.status(400).json({ error: e });
-  }
-  // At this point, assume the form data is completely valid.
-  try {
-    const { fullness, genres } = updateData;
-    const updatedLibrary = await libraryData.formUpdate(
-      libraryId,
-      fullness,
-      genres
-    );
-    res.json(updatedLibrary);
-  } catch (e) {
-    res.status(500).render('error', {errorCode: 500, title: "Error Page"});
-  }
-}).get(async (req, res) => {
-  let id = req.params.id;
-  
-  id = validation.checkValidId(id);
-
-  let library;
-
-  try {
-    library = await libraryData.get(id);
-  } catch (e) {
-    return res.status(404).send("Error: No library with given ID");
-  }
-
-  try {
-    res.render('libraries/library', { title: library.name, library: library, isLoggedIn: true });
-  } catch (e) {
-    res.status(500).send(e);
-  }
-});
-
-router.route("/fullnessForm").get(async (req, res) => {
-  // need to come back and fix the ID with cookie stuff
-  res.render("libraries/fullness", {
-    title: "Fullness Update Form",
-    id: "NEED TO FIX",
-  });
-});
-
-router
-  .route('/comments/create/:id')
-  .post(async (req, res) => {
-    //GET USER ID FROM MIDDLEWARE COOKIE
-    let id = req.params.id;
+    // need to come back and fix the ID with cookie stuff
+    try {
+      res.render("libraries/new", { title: "Creating a Library", id: "NEED TO FIX" });
+    } catch (error) {
+      
+    }
     
-    //let userId = req.user._id
-
-    id = validation.checkValidId(id);
-
-    let text = req.body.text;
-
-    text = validation.checkString(text);
-
-    let library;
-
+  })
+  .post(async (req, res, next) => {
+    // Submit the new library form page
     try {
-      library = await libraryData.get(id);
+      console.log(req.body)
+      // validation.checkImageFileString(req.body.image, "Libarys Image");
```

Comment:
Are we going to validate the image string in next PR or @DanielleDauphinais will handle it ?

Reply:
I think Danielle is taking care of that

Reply:
We can't validate the image string before inputing it to the database based on what I have tried to do so we will have to work on a delete function of the image in conjunction with validating the file. But I will work on this Evan just wanted to organize this full first